### PR TITLE
adding invokelatest

### PR DESCRIPTION
**Pull request by schlichtanders**
fix https://github.com/JuliaPluto/Malt.jl/issues/83

this actually worked :+1: 

Not sure whether this should need an extra test. The lines are of course tested by the current tests already.

### CODE REVIEW

**Comment by fonsp** [comment]
I wonder if this also affects performance... Let's see if the performance tests pass! Otherwise it might be good to look into, and maybe the Distrubted source code can help

**Comment by fonsp** [comment]
Can you update from `main`? I fixed the igthub aciton

**Comment by fonsp** [comment]
I also gave you write access to the repo

**Comment by schlichtanders** [comment]
> Can you update from `main`? I fixed the igthub aciton

I rebased :+1:

**Comment by schlichtanders** [comment]
impressive that all tests on older julia versions fail - is this normal?

File: src/shared.jl [hunk 1]
```diff
@@ -38,7 +38,7 @@ function _serialize_msg(io::IO, msg_type::UInt8, msg_id::MsgID, msg_data::Any)
     try
         write(io, msg_type)
         write(io, msg_id)
-        serialize(io, msg_data)
+        invokelatest(serialize, io, msg_data)
```
Comment (Pangoraw) [inline src/shared.jl:41]:
```suggestion
        Base.invokelatest(serialize, io, msg_data)
```

File: src/worker.jl [hunk 2]
```diff
@@ -75,7 +75,7 @@ function serve(server::Sockets.TCPServer)
         msg_id = read(io, MsgID)
         
         msg_data, success = try
-            (deserialize(io), true)
+            (invokelatest(deserialize, io), true)
```
Comment (Pangoraw) [inline src/worker.jl:78]:
```suggestion
            (Base.invokelatest(deserialize, io), true)
```

**Review by Pangoraw — APPROVED** [review]
Benchmark tests do not seem impacted :heavy_check_mark:

**Comment by schlichtanders** [comment]
Thank you @Pangoraw, merging this then now

**Comment by fonsp** [comment]
@schlichtanders Can you make a release?

**Pull request merged**